Lib/test/test_embed.py

```diff
@@ -695,16 +667,8 @@ def check_global_config(self, configs):
 
         self.assertEqual(configs['global_config'], expected)
 
-    def check_path_config(self, configs, expected):
```

Comment:
Are we losing any test coverage by no longer checking the path config explicitly?

Reply:
Shouldn't be - there's no specific codepaths for this anymore other than the `Py_Set*` tests. All the path config is in the regular init config now.